### ISSUE TITLE
USWDS-Site - Bug: Tooltip needs to announce "tooltip" for context

### ISSUE THREAD

**Issue by amycole501** [uswds/uswds-site#2381]
### Describe the bug

The [tooltip component ](https://designsystem.digital.gov/components/tooltip/) the word "tooltip" isn't announced as a label.

The element that serves as the tooltip container should have a role [tooltip](https://w3c.github.io/aria/#tooltip).

Screen reader users are not able to discern the purpose of elements with role="tooltip" that do not have an accessible name. Aria tooltip elements must have discernible text that clearly describes the destination, purpose, function, or action for screen reader users.

[From w3c](https://w3c.github.io/aria/#tooltip): Authors SHOULD ensure that elements with the [role](https://w3c.github.io/aria/#dfn-role) tooltip are referenced through the use of [aria-describedby](https://w3c.github.io/aria/#aria-describedby) before or at the time the tooltip is displayed.

### Steps to reproduce the bug

1. Go to the tooltip component page
2. Listen to the example buttons using assistive technology (in my case I used JAWS and NVDA)
3. You hear the tooltip wording announced ("show on top button; top") but it's not declaring it as a tooltip. Currently it sounds like the button is repeating itself.

### Expected Behavior

Listen to the [tooltip example](https://dequeuniversity.com/library/aria/tooltip) on Deque's website. You can hear "tooltip colon" then the tooltip wording.

Ideally on our component examples you would hear "Tooltip colon: Top. Show on top".

### System setup

PC, Chrome, JAWS, NVDA

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://designsystem.digital.gov/about/community/#community-conduct).
- [X] I checked the [current issues](https://github.com/uswds/uswds-site/issues) for duplicate bug reports.
